hydrosdk/application.py

```diff
@@ -264,13 +266,13 @@ def __init__(self, stages: List['ExecutionStage']) -> 'ExecutionGraph':
         """
         self.stages = stages
 
-    def _asdict(self) -> Dict[str, List['ExecutionStage']]:
+    def _asdict(self) -> Dict[str, List]:
```

Comment:
Why removed List type?

Reply:
`_asdict` is called recursively for each `ExecutionStage`, hence `ExecutionGraph._asdict()` produces `Dict[str, List[Dict]]`, not `Dict[str, List[ExecutionStage]]`